### DISCUSSION TITLE
What to Back Up?

### DISCUSSION THREAD

**Discussion by bagelwoof** [kanboard/kanboard#5421] [Q&A]
Is there a plugin that does backups? 

If not, what should I back up? 

I see that there's an option to do a DB export in settings, but I'm wondering what else I should grab. If I'm cobbling together my own solution, I'll probably do something with restic and Backrest, as I'm already doing that with another couple small servers. I'm using Kanboard for personal/household planning, and my DB is miniscule. 

Is it possible to do a DB export or save from the CLI? I'm using sqlite.

**Comment by fguillot** [comment]
> What to Back Up?

Your database and the file attachments stored by default in the data folder.

You can use the [`.backup` command](https://sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_) for Sqlite databases: 

```
sqlite3 db.sqlite ".backup 'backup_db_sqlite'"
```
